Resources/doc/index.rst

```diff
@@ -59,7 +59,7 @@ Example applications
 The following bundles/applications use the FOSRestBundle and can be used as a
 guideline:
 
-- The `LiipHelloBundle`_ provides several examples for the RestBundle.
+- The `LiipHelloBundle`_ provides several examples for the RestBundle. (UNMAINTAINED)
```

Comment:
please remove all examples except `FOSCommentBundle`. all the others are outdated and we don't need to list them here.